makefiles/pgrn-pgxs.mk

```diff
@@ -1,4 +1,4 @@
-REQUIRED_GROONGA_VERSION = 14.0.0
+REQUIRED_GROONGA_VERSION = 15.1.8
```

Comment:
We should use minimal required version instead of the latest version to use PGroonga widely.

Reply:
FYI: The `language_model_vectorize` was added at 14.1.0.
https://groonga.org/docs/news/14.html#release-14-1-0-2024-11-05

Reply:
> I thought it should include automatic downloads too!

Ahh I see. So I think the latest is better here.